src/fakeData/fakeFacilities.ts

```diff
@@ -0,0 +1,43 @@
+import * as gqlTypes from '../typeDefs/gqlTypes.js'
+import { faker, fakerJA } from '@faker-js/faker'
+
+export function generateRandomCreateFacilityInput(healthcareProfessionalIds?: string[])
+    : gqlTypes.CreateFacilityInput {
+    const sex = faker.person.sexType()
```

Comment:
We haven't discussed adding `sex` to the database. I'm assuming you added this to show whether the healthcare professional is a man, woman, etc. If that's the case, I think we should use `gender` instead.

Reply:
Agreed. No plans to add it at the moment, but gender would be appropriate. This is just the fake data interface so the data generated is consistent.